stable/jenkins/templates/config.yaml

```diff
@@ -24,7 +24,7 @@ data:
       <version>{{ .Values.master.tag }}</version>
 {{- end }}
       <numExecutors>{{ .Values.master.numExecutors }}</numExecutors>
-      <mode>NORMAL</mode>
+      <mode>{{ .Values.master.executorMode }}</mode>
```

Comment:
please also make this configurable in the JCasC [here](https://github.com/helm/charts/blob/bb503e51e0cd312a92bf480ff9ec0d48e6cc9879/stable/jenkins/templates/_helpers.tpl#L78)